### PR TITLE
Malf AI Powers Update

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Updates a number of Malf AI powers and adds a few more/removes some

- Removes useless fireproof core power
- Upgrade turrets cost reduced from 50 to 30
- Overload Machine cost increased from 15 to 20
- Override Machine cost increased from 15 to 30
 - Can once again over-load containment field generators
 - Copy mimics will slowly die if they lack a target to attack
- Re-adds hostile lockdown, with a few tweaks (now with 99% less lag) for 30 CPU
 - Single use and it automatically lifts after 90 seconds
- Adds in Break Fire and Air Alarms for 25 CPU and 50 CPU respectively
 - Broken fire alarms won't alarm if an area is overheated; broken air alarms allow you to access "flood" mode
 - AI no longer has "Flood" mode by default
- Adds in lip-reading power for 30 CPU
 - When activated, you'll be able to hear what people are saying in and around your camera location

:cl: Fox McCloud
tweak: Removes fireproof core malf AI power
tweak: upgrade turrets malf AI power CPU reduced from 50 to to 30
tweak: Overload machine cost increased from 15 to 20
tweak: Override machine cost increased from 15 to 30
tweak: can override/overload field generators again
tweak: Machine uprisers will slowly die if they aren't attacking someone
add: Adds in Hostile lockdown malf AI power for 30 CPU
add: Adds break air and fire alarms malf AI powers for 50 and 25 CPU respectively
tweak: AI's no longer have flood mode on air alarms by default
add: Adds in lip reading malf AI power for 30 CPU
/:cl:

### CODE REVIEW

File: code/game/machinery/doors/airlock_control.dm [hunk 1]
```diff
@@ -10,8 +10,7 @@ obj/machinery/door/airlock
 
 obj/machinery/door/airlock/process()
 	if(arePowerSystemsOn() && cur_command)
-		spawn()
```
Comment (Fox-McCloud) [inline code/game/machinery/doors/airlock_control.dm]: no longer needed as `process` has `set waitfor = 0`

**Comment by BiberDark** [comment]
In my opinion this is a nerv again. The turrets are to  weak to be usefull. If they are in the core you have already lost. Theo only  usefull overload ans override is now more expensiv. Lockdown limited and onetime ?... First thing people do is cutting cams so it is indeed nice but not so helpfull afterall.

**Comment by Spacemanspark** [comment]
AIs already get destroyed incredibly easy. Especially during malf rounds. I'm a little concerned that this'll make it worse.

File: code/game/machinery/firealarm.dm [hunk 2]
```diff
@@ -49,10 +49,17 @@ FIRE ALARM
 	else
 		icon_state = "fire0"
 
+/obj/machinery/firealarm/emag_act(mob/user)
+	if(!emagged)
+		emagged = 1
+		if(user)
+			user.visible_message("<span class='warning'>Sparks fly out of the [src]!</span>",
+								"<span class='notice'>You emag [src], disabling its thermal sensors.</span>")
+		playsound(src.loc, 'sound/effects/sparks4.ogg', 50, 1)
```
Comment (tigercat2000) [inline code/game/machinery/firealarm.dm]: src.

File: code/game/gamemodes/malfunction/Malf_Modules.dm [hunk 3]
```diff
@@ -450,6 +525,4 @@ rcd light flash thingy on matter drain
 		if(href_list["showdesc"])
 			if(AM.mod_pick_name == href_list["showdesc"])
 				temp = AM.description
-
-	src.use(usr)
-	return
+	src.use(usr)
```
Comment (tigercat2000) [inline code/game/gamemodes/malfunction/Malf_Modules.dm]: src.

**Comment by Davdichan** [comment]
The core can definitely use some defense boosts, but if you get caught before you're ready to fight that's probably on the AI for fucking up. With the recent genetics nerf I haven't seen hulk rampages on the core recently, which was the biggest fear an AI had since a spaceproof hulk can not be contained or dealt with.

**Comment by Spacemanspark** [comment]
The new meta is thermite now. And you can guarantee that at least one chemist/ scientist made it at round start.

**Comment by Davdichan** [comment]
Shame we can't code thermite to require oxygen to burn.

**Comment by pinatacolada** [comment]
people are going to meta their ass out if they notice an overpressured or a plasma flood ¯\_(ツ)_/¯

**Comment by Earthdivine** [comment]
I don't see how any of this is a nerf? Override trumped Overload in every single instance, there was no reason to ever use Overloard.

Lockdown is insanely powerful, leaving only Pods and people already outside to be the only ones to hurt the AI

**Comment by Regen1** [comment]
>Lockdown is insanely powerful, leaving only Pods and people already outside to be the only ones to hurt the AI

Or, you know, anyone with a screwdriver, a wirecutter and some gloves

**Comment by Doukhan** [comment]
Now malf AIs are even more weaksauce, they _barely_ win, if anything they need a buff.

Sure a competent one can get themselves on station hidden somewhere/fortify the core but any half competent crew can lol thermite their way in in 30 seconds and ion their asses, the balance of power is very firmly against what is supposed to be the antagonist.

:-1:  for an even less lethal gamemode.

**Comment by Spacemanspark** [comment]
Actually hiding yourself on station doesn't help you unless you space the pinpointer.

**Comment by Doukhan** [comment]
@Spacemanspark The amount of times I've seen captains break into an "empty" AI core, only to declare a station wide manhunt when the AI is 3 squares away behind one wall in a side alcove...

Getting yourself on station is still a great advantage, a borg dragging the core between a couple of secure locations always screws over any murder attempt.

**Comment by TheDZD** [comment]
Also somewhat concerned about the AI being too easy to kill, although I don't think this is a problem exclusive to malf, considering malf AIs can shunt (although they rarely ever do). Reducing the cost of the turret upgrade might help a bit, but overall the AI core is just too easy to break into.

**Comment by BiberDark** [comment]
And so it got merged ? What is with traitor ai ? Maybe more turrets and a safer power supply ?

**Pull request merged**